src/client/components/RobotLogo/RobotLogo.stories.js

```diff
@@ -5,10 +5,6 @@ import RobotLogo from './RobotLogo';
 export default {
   title: 'Components / Robot Component',
   component: RobotLogo,
-  argTypes: {
-    onClick: { action: 'clicked' },
```

Comment:
This should change to handleOnClick

Reply:
when i am doing this storybook is not working for me

Reply:
its done now